spec/support/capybara.rb

```diff
@@ -9,6 +9,7 @@
   browser_options.args << '--headless'
   browser_options.args << '--disable-gpu'
   browser_options.args << '--no-sandbox'
+  browser_options.args << '--disable-dev-shm-usage'
```

Comment:
All capybara tests fail in Docker without this configuration.
More info:  https://developers.google.com/web/tools/puppeteer/troubleshooting#tips

Tests are passing now, though I think it does cause one specific test to fail:
```
1) Canceling Account Creation From the enter password page redirects to the branded start page
     Failure/Error:
       expect(current_url).to eq \
         new_user_session_url(request_id: ServiceProviderRequestProxy.last.uuid)

       expected: "http://www.example.com/?request_id=40a7d906-f37c-441f-951b-9bca25091c6e"
            got: "http://www.example.com/?request_id=6a6d76b0-3707-440c-bc4b-36045b8d4f0f"

       (compared using ==)
     # ./spec/features/saml/ial1/account_creation_spec.rb:30:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:82:in `block (2 levels) in <top (required)>'
```

Reply:
So I'm seeing different errors on different runs.  Could use an assist at some point to see if others see the same / know what's going on

```
Failures:

  1) throttling requests high requests per ip when the number of requests is higher than the limit throttles with a custom response
     Failure/Error: expect(response.status).to eq(429)

       expected: 429
            got: 200

       (compared using ==)
     # ./spec/requests/rack_attack_spec.rb:70:in `block (4 levels) in <top (required)>'
```

Reply:
Taking a look!

Reply:
These failures all seem intermittent :/